packages/sdk/chainDeploy/mainnets/polygon.ts

```diff
@@ -558,9 +558,13 @@ export const deploy = async ({ run, ethers, getNamedAccounts, deployments }: Cha
 
   /// Addresses Provider - set bUSD
   const addressesProvider = (await ethers.getContract("AddressesProvider", deployer)) as AddressesProvider;
-  const tx = await addressesProvider.setAddress("bUSD", assets.find((a) => a.symbol === assetSymbols.BUSD)!.underlying);
-  await tx.wait();
-  console.log("setAddress: ", tx.hash);
+  const busdAddress = assets.find((a) => a.symbol === assetSymbols.BUSD)!.underlying;
```

Comment:
why do we store bUSD on polygon?

Reply:
I have idea, I thought you added it

Reply:
let's keep it as I think we use it in the tests